driver/test_helpers.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 // initKubeFake initialize the Kube clientset with a fake
-func (kd *KubeDriver) initKubeFake(objects ...runtime.Object) {
+func initKubeFake(driver *KubeDriver, objects ...runtime.Object) {
```

Comment:
Could we please name that argument `kd` instead of `driver`? It is easier to disambiguate between a `driver` variable, and the `driver` package in this manner...
***
This is simply a question of convenience. I realize that within the `driver` package, there is no name conflict.

Reply:
good point. i was also trying to distinguish it from the package global `kd`

Reply:
Perhaps something other than `kd` and `driver` ... like `kDriver`, or `kubeDriver`?